kube/services/jobs/covid19-etl-job.yaml

```diff
@@ -10,7 +10,7 @@ spec:
         app: gen3job
     spec:
       containers:
-      - name: tube
+      - name: covid19-etl
```

Comment:
can we maybe rename this one covid19-jhu-etl-job.yaml?

Reply:
I have separate PR to have consistent naming: https://github.com/uc-cdis/cloud-automation/pull/1163